### DISCUSSION TITLE
error 401 MSG_LOGIN_UNAUTHORIZED_ERROR

### DISCUSSION THREAD

**Discussion by lgmg** [kubernetes/dashboard#9167] [Q&A] [answered]
hello, I am struggling with this error on the auth API. And still cannot login.
I am on a baremetal cluster, behind a router with haproxy and with a haproxy ingress controller
 On the auth pod log i have :
E0617 11:10:20.199174       1 handler.go:33] "Could not get user" err="MSG_LOGIN_UNAUTHORIZED_ERROR"
it was as if it did not receive the token. I tried to call the auth /api/v1/me service directly with a curl from the control plane with a authorization header containing a token but I still have this error. Could you provide me with an example of a direct call to the auth service ?

**Comment by lgmg** [comment, marked as answer]
update : there was a pb with my token. A newly generated one worked

**Comment by lgmg** [comment]
in fact, the dashboard is still not login, but a direct call to the auth service works. is there a way to trace the request header through all the layers ? I suspect that one of the proxy is not transmitting correctly the bearer token. 

In fact, with requestly, it works perfectly, this means that proxy is ok

**Reply by WangYihang** [reply]
I also encountered the same problem, and I tried open dashboard on the private tab, it works.

So this problem can be solved it by clearing the follow browser cookies.

* `token`
* `refresh_token`

<img width="444" alt="image" src="https://github.com/user-attachments/assets/e6f65a5d-aba4-42ec-82f0-430d8da0a8b1">


After cleaning the token, I can login sucessfully.

Hope it helps.
